File: vidralglass/src/app/model/venda.ts
export class Venda {
    id!: number;
    status!: string;
    dataVenda!: Date;
    dataEntrega!: Date;

    constructor(){
        this.id = 0;
        this.status = "";
        //this.dataVenda = Date();
        //this.dataEntrega = Date();
        
        }
}

File: api/src/main/java/br/cefet/vidralglass/model/Venda.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package br.cefet.vidralglassbase.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import lombok.experimental.SuperBuilder;

/**
 *
 * @author criad
 */
@Data
@Getter
@Setter
@ToString
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class Venda {
    protected int id;
    protected String status;
    protected String dataVenda;
    protected String dataEntrega;
}
